packages/react/index.js

```diff
@@ -3,6 +3,8 @@
 exports.internal = {
   babelPlugin: require('./lib/babel'),
   template: require('./lib/template'),
+  runtime: require('./lib/runtime'),
 };
+exports.internal.babelPlugin.path = require.resolve('./lib/babel');
```

Comment:
If we add some property to `babelPlugin`, I would prefer it happening in `./lib/babel`.